tests/models/conftest.py

```diff
@@ -63,3 +63,49 @@ def starting_links() -> dict:
         "aggregate": "test",
         "no_aggregate_reason": "This is a test database",
     }
+
+
+@pytest.fixture(scope="function")
+def good_structure() -> dict:
+    """Returns a 'good' structure that does not need to mapped, which can be
+    deformed to test different validators.
+
+    """
+    import datetime
+
+    return {
+        "id": "db/1234567",
+        "type": "structures",
+        "attributes": {
+            "last_modified": datetime.datetime.now(),
+            "elements": ["Ge", "Si"],
+            "nsites": 3,
+            "nelements": 2,
+            "elements_ratios": [0.5, 0.5],
+            "chemical_formula_reduced": "GeSi",
+            "chemical_formula_hill": "GeSi",
```

Comment:
Since fields are not cross-validated, I've added an extra test case with both C and H in them as "deformities". Do you have any opinions on my second point above? Should we add pairwise validators for all correlated fields?

Reply:
> Since fields are not cross-validated, I've added an extra test case with both C and H in them as "deformities". Do you have any opinions on my second point above? Should we add pairwise validators for all correlated fields?